Library/ViewModels/PledgeViewModel.swift

```diff
@@ -860,38 +860,75 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
         )
       }
 
-    Signal
-      .combineLatest(
-        createBackingData,
-        baseReward,
-        additionalPledgeAmount,
-        allRewardsShippingTotal
+    // Emmits value when Pledge Button is clicked.
+
+    Signal.combineLatest(
+      createBackingData,
+      baseReward,
+      additionalPledgeAmount,
+      allRewardsShippingTotal
+    )
```

Comment:
I think we can assign this to a constant called `eventSignal` or something similar because we are reusing it twice here.

Reply:
That works, I'll add that change. Thanks.